README.md

```diff
@@ -958,17 +934,33 @@ targets:
 
 Here is how you can integrate your GitHub project with `craft`:
 
-- Enable your project in Zeus: https://zeus.ci/settings/github/repos
-- Configure your CI systems (Travis, AppVeyor, etc.) to send build artifacts to Zeus
-  - Allow building release branches (their names follow `release/{VERSION}` by
-    default, configurable through `releaseBranchPrefix`)
-  - Add ZEUS_HOOK_BASE as protected to CI environment
-- Add `.craft.yml` configuration file to your project
+1. Set up a workflow that builds your assets and runs your tests. Allow building
+   release branches (their names follow `release/{VERSION}` by default,
+   configurable through `releaseBranchPrefix`).
+2. Use the official `actions/upload-artifact@v2` action to upload your assets.
+   Here is an example config (step) of an archive job:
+
+    ```yaml
```

Comment:
Indentation is one off, does it copy/paste fine from the rendering?

Reply:
Possible. Will fix.